kustomize/overlays/prod/patches/deploy-config.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 data:
-  EVENT_BRIDGE_EXECUTOR_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/executor:cba22b67e849d7dcdb06a510d4c37a857fa4d74a-jvm
-  EVENT_BRIDGE_ISTIO_GATEWAY_NAME: istio-ingressgateway
+  EVENT_BRIDGE_EXECUTOR_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/executor:8b9275b36f8ceac5b600d0afd852e5cde8023248-jvm
+  EVENT_BRIDGE_ISTIO_GATEWAY_NAME: rhose-ingressgateway
```

Comment:
@r00ta Can you double-check with @vaibhavjainwiz where/how this is set for the AddOn build of our operator.

cheers,

Rob

Reply:
Once this PR would merge I would update these values

Reply:
Thanks @vaibhavjainwiz